### PR TITLE
Wc view

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: src/view/wc.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <style>
+    .wrapper { text-align: center }
+    .dice { font-size: 200px }
+    .roll { font-size: 50px;}
+  </style>
+  <div class="wrapper">
+    <div class="dice"></div>
+    <button class="roll"> Roll </button>
+  </div>
+`
```
Comment (tylerbutler) [inline src/view/wc.js:20]: Pretty!

Comment (micahgodbolt) [reply src/view/wc.js:20]: now to try this on the JS view :D

File: src/view/wc.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+export const wcRenderView = (diceMap, elem) => {
+
+  customElements.define("wc-dice", Dice);
+
+  const dice = document.createElement("wc-dice");
+
+  dice.onRoll = number => diceMap.set(diceValueKey, number);
+
```
Comment (tylerbutler) [inline src/view/wc.js:54]: nit: all the empty lines seem unnecessary...

**Review by tylerbutler — APPROVED** [review]

File: src/view/wc.js (hunk 2 above)
Comment (micahgodbolt) [reply src/view/wc.js:54]: yeah, was breaking things up thematically, but each theme got really small in the end :D

Comment (tylerbutler) [reply src/view/wc.js:54]: hahaha been there! :)

Comment (micahgodbolt) [reply src/view/wc.js:54]:
```suggestion
  customElements.define("wc-dice", Dice);
  const dice = document.createElement("wc-dice");
  dice.onRoll = number => diceMap.set(diceValueKey, number);
```

**Pull request merged**